backend/orchestrator/functions/requestSyncDeployment/handler.ts

```diff
@@ -0,0 +1,19 @@
+import { requestSyncDeployment } from '@swarmion/orchestrator-contracts';
+import { applyHttpMiddlewares } from '@swarmion/serverless-helpers';
+
+import ServiceEventEntity from 'libs/models/serviceEvent';
+
+const handler = requestSyncDeployment.handler(async event => {
+  const { serviceId, applicationId } = event.body;
+
+  const timestamp = new Date().toISOString();
```

Comment:
Per my previous comment, we would not need to manually create this

Reply:
I replaced it by an ulid which is stronger than a timestamp